draft-kuehlewind-taps-crypto-sep.md

```diff
@@ -328,6 +339,20 @@ transition path to enable easy deployment of new protocols.
 
 <!-- CAW: a hint to TLPN could be dropped here -->
 
+# Transport Service Architecture Integration
+
+Pauly et al. {{I-D.pauly-taps-arch}} describe the architecture of a generic
```

Comment:
Maybe:

The Transport Services Architecture ({{I-D.pauly-taps-arch}}) describes a system that can provide transport security functionality behind a common interface.

Reply:
I like it! Adopted.